client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/email-provider-generic-form.tsx

```diff
@@ -0,0 +1,868 @@
+import { Button, Gridicon } from '@automattic/components';
+import { MinimalRequestCartProduct } from '@automattic/shopping-cart';
+import classNames from 'classnames';
+import emailValidator from 'email-validator';
+import { translate, TranslateResult, useRtl, useTranslate } from 'i18n-calypso';
+import { countBy, find, groupBy, includes, map, mapValues } from 'lodash';
+import React, { ChangeEvent, Fragment, FunctionComponent, ReactNode, useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import FormFieldset from 'calypso/components/forms/form-fieldset';
+import FormInputValidation from 'calypso/components/forms/form-input-validation';
+import FormLabel from 'calypso/components/forms/form-label';
+import FormPasswordInput from 'calypso/components/forms/form-password-input';
+import FormSelect from 'calypso/components/forms/form-select';
+import FormTextInput from 'calypso/components/forms/form-text-input';
+import FormTextInputWithAffixes from 'calypso/components/forms/form-text-input-with-affixes';
+import { googleApps, googleAppsExtraLicenses } from 'calypso/lib/cart-values/cart-items';
+import {
+	getGSuiteMailboxCount,
+	hasGSuiteWithUs,
+	isGoogleWorkspaceProductSlug,
+	isGSuiteProductSlug,
+} from 'calypso/lib/gsuite';
+import { GSuiteNewUser, GSuiteProductUser } from 'calypso/lib/gsuite/new-users';
```

Comment:
I think these should be type imports.
```suggestion
import type { GSuiteNewUser, GSuiteProductUser } from 'calypso/lib/gsuite/new-users';
```